### PR TITLE
[General Reorganization] Computer Hardware

### PR DESCRIPTION
**Pull request by caupolicandiaz**
### Description

Moved the following entries under Computer Hardware:
-CPU
-Processor
-RAM

### Type of Change

- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [ x] All writings are my own.
- [ x] My entry follows the Codecademy Docs style guide.
- [ x] My changes generate no new warnings.
- [ x] I have performed a self-review of my own writing and code.
- [ x] I have checked my entry and corrected any misspellings.
- [x ] I have made corresponding changes to the documentation if needed.
- [x ] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x ] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [ x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Review by SSwiniarski — APPROVED** [review]
@caupolicandiaz. Looks fine, but did we resolve the attribution when files were moved, or am I misremembering the problem?

**Comment by caupolicandiaz** [comment]
> @caupolicandiaz. Looks fine, but did we resolve the attribution when files were moved, or am I misremembering the problem?

As far as I remember Lucas reported that it should no longer be an issue. But, it's probably best to regularly check and make sure things are working as they should.

**Review by KTom101 — COMMENTED** [review]
Only see a minor change.

File: content/general/concepts/computer-hardware/computer-hardware.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  - 'Computer Science'
+  - 'Code Foundations'
+Tags:
+  - 'Infrastructure'
+CatalogContent:
+  - 'paths/computer-science'
+  - 'paths/code-foundations'
+---
+
+**Computer hardware** represents the physical technologies, the infrastructure, that enables the processing and other functionalities that modern devices provide. Advances in computer hardware are part-and-parcel to the growth and proliferation of computing overall.
```
Comment (yangc95) [inline content/general/concepts/computer-hardware/computer-hardware.md:14]:
```suggestion
**Computer hardware** represents the physical technologies of the infrastructure, that enables the processing and other functionalities that modern devices provide. Advances in computer hardware are part-and-parcel to the growth and proliferation of computing overall.
```

**Review by yangc95 — APPROVED** [review]

**Comment by yangc95** [comment]
Live links: General [Computer Hardware](https://www.codecademy.com/resources/docs/general/computer-hardware): [CPU](https://www.codecademy.com/resources/docs/general/computer-hardware/cpu), [Processor](https://www.codecademy.com/resources/docs/general/computer-hardware/processor), [RAM](https://www.codecademy.com/resources/docs/general/computer-hardware/ram)

**Pull request merged**